rocketmq-doc/_posts/2025-02-16-release-v0.4.0.md

```
@@ -0,0 +1,723 @@
+      url: "https://github.com/mxsm/rocketmq-rust/releases/tag/v0.4.0"
+---
+
+## New Feature and improve performance
+
+- ✨Fix some bugs
+- 🌐[ Add rocketmq-rust doc web site](https://rocketmq-rust.ljbmxsm.com/)
+- 🚀 Add new crates: rocketmq-tui and rocketmq-tools
+- 🚀 Add some new features
+
+## What's Changed
+* [ISSUE #1210]✨Update the version to 0.4.0 for development by @mxsm in https://github.com/mxsm/rocketmq-rust/pull/1211
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)